perun-core/src/main/java/cz/metacentrum/perun/core/bl/VosManagerBl.java

```diff
@@ -161,14 +161,14 @@
 
 	/**
 	 * Finds MemberCandidates who can join the Group. If the given vo is not null, it searches only
-	 * users who belong to this Vo.
+	 * users who belong to this Vo or who have ues in any of given extSources.
 	 *
 	 * @param sess session
-	 * @param vo vo if is null, users are searched in whole perun, otherwise only members of this vo are used.
+	 * @param vo vo if is null, users are searched in whole perun, otherwise members of this vo are used and users with ues in given extSources.
```

Comment:
This sentence is hard to understand. What exactly happened if there is not an empty list of ext sources?

Reply:
I've edited the javadoc.